build.sbt

```diff
@@ -88,6 +88,13 @@ val commonSettings = Sonatype.sonatypeSettings ++ assemblySettings ++ Seq(
 
   scalastyleSources in Compile ++= (unmanagedSourceDirectories in Test).value,
   testOptions += Tests.Argument(TestFrameworks.JUnit, "-q", "-v"),
+  testOptions ++= {
+    if (sys.env.contains("SLOW")) {
```

Comment:
This isn't set on circleCI right? Wonder if we should set up separate IT builds (nightly / pre-release)?

Reply:
Added a step in release guide to do this manually.